### PR TITLE
Fix generation of schemas for JSON array columns

### PR DESCRIPTION
**Pull request by erizocosmico**

### CODE REVIEW

File: generator/template.go [hunk 1]
```diff
@@ -211,14 +216,23 @@ func (td *TemplateData) GenSubSchemas() string {
...
+
 	td.genSubschemaFieldsInit(buf, parent, f.Fields, "fmt.Sprint(n)")
 	buf.WriteString("}\n}\n\n")
 }
 
 // genSchemaPath generates the path needed to access the given field in JSON.
 // If prependLast is given, it will add it before the last element of the path.
-func (td *TemplateData) genSchemaPath(f *Field, prependLast string) string {
+// It also returns a boolean reporting whether the path has a single level of
+// depth (that is, we're talking about the column itself.
```
Comment (dpordomingo) [inline generator/template.go]: end parenthesis  missing

File: generator/template.go [hunk 2]
```diff
@@ -251,12 +267,14 @@ func (td *TemplateData) genSubschemaFieldsInit(buf *bytes.Buffer, parent string,
 			if f.IsJSON && len(f.Fields) > 0 {
 				td.genSubschemaInit(buf, parent, f)
 			} else if strings.HasPrefix(f.Type, "[") {
```
Comment (dpordomingo) [inline generator/template.go]: it could be used the already defined `isSliceOrArray(f)`

Comment (erizocosmico) [inline generator/template.go]: yes

**Comment by erizocosmico** [comment]
Fixed @dpordomingo

**Comment by dpordomingo** [comment]
If your last --amend commit only changes what I proposed, LGTM

**Comment by erizocosmico** [comment]
yes, it does

**Pull request merged**